src/EditorFeatures/Test2/Diagnostics/DiagnosticServiceTests.vb

```diff
@@ -1917,6 +1918,9 @@ class MyClass
                        </Workspace>
 
             Using workspace = TestWorkspace.CreateWorkspace(test)
+                ' set csharp closed diagnostic option on
+                workspace.Services.GetService(Of IOptionService)().SetOptions(workspace.Options.WithChangedOption(ServiceFeatureOnOffOptions.ClosedFileDiagnostic, LanguageNames.CSharp, True))
```

Comment:
You can write workspace.Options = workspace.Options.WithChangedOption(). Turns out we made that settable. (I know this only because I'm changing everybody who is doing this...:smile:)